manim/mobject/mobject.py

```diff
@@ -511,7 +519,9 @@ def __isub__(self, other):
     def set(self, **kwargs) -> "Mobject":
         """Sets attributes.
 
-        Mainly to be used along with :attr:`animate` to
+        I.e. ``my_mobject.set(foo=1)`` applies ``my_mobject.foo = 1``.
```

Comment:
Like with animate I felt an inline example was necessary as the method signature doesn't give anything away.

Reply:
👍